_posts/2024/2024-01-15-the-first-five-questions.md

```diff
@@ -0,0 +1,110 @@
+What should the motivating questions for each episode be?
+I've listed four below,
+and I'd like to set up an hour-long call later this month
+for anyone who's interested in expanding on them,
+replacing them with better ones,
+or suggesting another approach entirely.
+If you'd like to take part,
+please [email me](mailto:{{site.author.email}}) and I'll send you an invitation.
+
+Why do we have so many meetings?
+:   Coordination costs rise as the square of the number of people,
+    so large groups must (a)
+    make work uniform
+    so that sets of people are effectively interchangeable (e.g., assembly lines and typing pools),
+    (b) create sub-groups and invest effort in coordinating their work,
+    or (c) allow sub-groups to work independently and pay the cost of duplicated or contradictory effort.
+
+Why do so many meetings suck?
+:   Most people don't seek to maximize profit, but to maximize social standing within their peer group.
+    Some people do this by being helpful, but others try to do it by establishing social dominance.
+    Their meetings turn into one-up displays,
+    which often take the form of interrupting people they perceive as having lower social status.
+    Social norms (such as meeting moderation rules) exist in part to constrain such behavior.
+
+How should software developers be assessed?
+:   Exams and performance reviews are facts of life in academia and industry.
+    They exist for good reasons—some people truly are more skilled or more productive than others—but
+    every quantitative metric for comparing people will inevitably be gamed,
+    while every qualitative metric is subject to various biases.
+    One response is to allow people to define their own victory conditions,
+    but that makes it difficult or impossible to compare people.
+
+Who owns the software I write?
+:   Depending on your circumstances,
+    the software you write may belong to your employer
+    even if you wrote it on your own time using your own computer
+    (because they provided the training you needed to write it),
+    or they may only have an interest in it if you used their physical resources,
+    or it may be entirely yours as long as you wrote it on your own time.
+    The answer depends on your venue's rules about intellectual property;
+    that concept is less than two hundred years old,
+    but has expanded steadily over the last century
+    via processes such as regulatory capture
+    in ways that further the interests of profit-driven corporations.
+
+Why do hairdressers have to be licensed but programmers don't?
+:   First,
+    notice how the question uses the social status of the two professions:
+    it could be rewritten as,
+    "Isn't it ridiculous that lowly hairdressers have to get licenses
+    but exalted programmers don't?"
+    The answer is that licensure is tied to professional responsibility
+    and to the way in which middle-class professions such as law and medicine
```

Comment:
although law and medicine do require licensure and continuing education training standards. So I wonder if the question here is actually more about "Why don't programmers globally require licensure when so many other professions do?"